modules/light-clients/08-wasm/keeper/msg_server.go

```diff
@@ -41,3 +43,55 @@ func (k Keeper) PushNewWasmCode(goCtx context.Context, msg *types.MsgPushNewWasm
 		CodeId: codeID,
 	}, nil
 }
+
+func (k Keeper) ClientStore(ctx sdk.Context, clientID string) sdk.KVStore {
+	clientPrefix := []byte(fmt.Sprintf("%s/%s/", host.KeyClientStorePrefix, clientID))
+	return prefix.NewStore(ctx.KVStore(k.storeKey), clientPrefix)
+}
```

Comment:
I don't see why we need function. Why do we need this?

Reply:
Removed